.github/workflows/unsafe_app_ci.yml

```diff
@@ -165,6 +165,7 @@ jobs:
         run: |
           fvm flutter build web \
             --release \
+            --dart-define DEVELOPMENT_STAGE=ALPHA \
```

Comment:
WDYT? We could also use sth like "PREVIEW" here. We need this so that the chips are shown in the preview app for GitHub PRs.

Reply:
Doesn't work...

Reply:
Add a new stage like "PREVIEW" sounds good 👍